docs/install.md

```diff
@@ -58,12 +58,6 @@ helm install kubedriver-<version>.tgz --name kubedriver -f custom-values.yml
 
 The driver runs with SSL enabled by default. The installation will generate a self-signed certificate and key by default, adding them to the Kubernetes secret "kubedriver-tls". To use a custom certificate and key in your own secret, override the properties under "apps.config.security.ssl.secret".
 
-### Confirm 
-
-You can confirm the driver is working by accessing the Swagger UI included to render the API definitions.
-
-Access the UI at `https://your_host:31684/api/driver/ui` e.g. [`http://localhost:31684/api/driver/ui`](http://localhost:31684/api/driver/ui)
-
```

Comment:
Does the user need another way to confirm the driver is working now that the swagger ui has been removed?

Reply:
@hhellyer Only through `oc get pod kubedrivexxxxx` will show the status of the pod and no other endpoints are available to verify the same.

Reply:
This driver uses the Ignition framework which, after the removal of the Swagger UI, has suggested using the `/management/health` endpoint instead: https://github.com/IBM/ignition/blob/develop/ignition/templates/driver_template/docs/index.md#install

So all drivers should be updated to suggest the same.

Reply:
@dvaccarosenna @hhellyer 
Added ```http://<kubernetes-node-ip>:{(helm.node_port)}/management/health```

Reply:
`{(helm.node_port)}` can be replaced with `31684` as we know that's the default node port